app/main/authorize/access_token_sign_in_required.py

```diff
@@ -82,8 +107,13 @@ def decorated_view(*args, **kwargs):
 def _validate_or_refresh_tokens(access_token, refresh_token):
     tokens_are_refreshed = False
 
-    if not (access_token and refresh_token):
-        raise InvalidAccessToken
+    # PERFORMANCE TEST MODE
+    if os.getenv("PERF_TEST", "False") == "True":
```

Comment:
An idea: we could have a generic 'test mode' where endpoints accept an authorization header in the form of the token, this way we don't have to embed logic or new env variables for each test mode (e.g. accessibility tests in the future). Overall, this is a really good piece of functionality, its gonna make our lives much easier testing pages.